mibidata/tests/test_mibi_image.py

```diff
@@ -112,16 +109,11 @@ def test_set_channels_invalid_tuple(self):
         with self.assertRaises(ValueError):
             image.channels = invalid_tuple_3
 
-    def test_convert_from_previous_metadata_versions(self):
-        image = mi.MibiImage(TEST_DATA, TUPLE_LABELS)
-        image.point_name = OLD_METADATA['point_name']
-        image.folder = OLD_METADATA['folder']
-        image.MIBItiff_version = None
+    def test_convert_from_previous(self):
```

Comment:
Good point that it's redundant, but the previous version added attributes to the MibiImage instance and then had to workaround the fact that they were inconsistent with what was established when constructing the instance. Since `_convert_from_previous` is a private method I'd rather test it in the way it actually gets used (as part of the constructor) rather than in an artificial way. We can remove this test if it's a duplicate at this point.

Reply:
I know, I was mocking the construction in order to specifically test the function. Usually each function (even if it's protected) should have its test, otherwise, if something changes in the constructor (i.e. a loop around the convert function), it could be easily missed by the test and pass unnoticed, diminishing the coverage and leaving an important part of the software untested.